src/sentry/static/sentry/app/components/organizations/headerItem.jsx

```diff
@@ -28,22 +28,12 @@ class HeaderItem extends React.Component {
     this.props.onClear();
   };
 
-  handleChevronClick = e => {
```

Comment:
Is this something we want to remove? Will env selector and date selector also have "apply" buttons?

Reply:
hmm yeah good point. they probably should; if we prefer it here why wouldn't we elsewhere? I'll look into it. 

We can either keep them alive for the other ones, or remove them and have an awkward in-between period, or I can try to include submit buttons for all three here.

Reply:
Makes sense to add apply to env selector -- date selector has a different layout and might be a bit trickier? I think ideally we would have it on all 3.